egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -194,6 +221,23 @@ def PrintConfig(file_name, config_lines):
     f.write("\n".join(config_lines['component-nodes']))
     f.close()
 
+def ParseCnnString(cnn_string):
```

Comment:
I think you should allow for specification of default parameters. It would be easy to do this using argparse.